code/datums/components/thermo.dm

```diff
@@ -0,0 +1,78 @@
+ // This is basicly a component that acts like a space heater
+ // Like something thatt emits heat like a flare or absorbes heat like ice
+ // You can set up ranges for when it starts smoking or when it catches fire or
+ //  when it ices over and frezes over, but there is no implmention of sever cold yet
+// Most of it is copyed over from the space_heater code
+/datum/component/thermo
```

Comment:
Use autodoc for your component as described here https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md